scannergradle.properties

```
@@ -6,8 +6,13 @@ organizationUrl=https://www.sonarsource.com
 homepageUrl=https://docs.sonarqube.org/latest/analysis/scan/sonarscanner-for-gradle/
 issueTrackerUrl=https://jira.sonarsource.com/browse/SONARGRADL/issues
 sourcesUrl=https://github.com/SonarSource/sonar-scanner-gradle
-archivedVersions=2.8,3.0,3.1,3.1.1,3.2
-publicVersions=3.3
+archivedVersions=2.8,3.0,3.1,3.1.1,3.2,3.3
+publicVersions=3.4.0.2513
+
+3.4.0.2513.description=Compatible with Gradle 8 and Java 17
+3.4.0.2513.date=2022-06-08
+3.4.0.2513.changelogUrl=https://jira.sonarsource.com/secure/ReleaseNote.jspa?projectId=10974&version=16991
+3.4.0.2513.downloadUrl=https://plugins.gradle.org/plugin/org.sonarqube
```

> Compatible with Gradle 8 and Java 17

Was it really not supported before?

Also, some suggestions below:
```suggestion
publicVersions=3.4

3.4.description=Support Gradle 8 and Java 17, increase socket connect timeout to 30s
3.4.date=2022-06-08
3.4.changelogUrl=https://jira.sonarsource.com/secure/ReleaseNote.jspa?projectId=10974&version=16991
3.4.downloadUrl=https://plugins.gradle.org/plugin/org.sonarqube
```